nextflow/nextflow_examples.ipynb

```diff
@@ -0,0 +1,1169 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a7622c4-2c5b-45f7-a638-f437e8a21f1b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!nextflow clean -n <RUN_NAME>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1d54a69-3bd8-4127-a477-da64f64baf36",
+   "metadata": {},
+   "source": [
+    "If you're satisfied with deleting the pipeline working directories indicated above, replace `<RUN_NAME>` with the name of the desired pipeline execution in the cell below, then run the cell to permanently remove files associated with that run. \n",
+    "\n",
+    "If you'd prefer to clean up directories associated with all previous runs, you can replace `<RUN_NAME>` with `$(nextflow log -q)`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3112675b-f728-445f-84c5-041f169cd76b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!nextflow clean -f <RUN_NAME>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fbfec09-7790-4956-a807-08af4a250397",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Example 2: Run an `nf-core` workflow\n",
+    "\n",
+    "Nextflow's `nf-core` is a curated collection of validated Nextflow pipelines, most of which can be run on almost any computing environment, including your Terra cloud environment. These pipelines are subject to transparent versioning and each version is run against automated testing prior to release. In this exercise, we will perform RNASeq on by running the nf-core/RNASeq pipeline from the [`nf_core` collection](https://nf-co.re/pipelines). The cost of running this example will typically be no more than $1.\n",
+    "\n",
+    "If desired, you can preview a fully-executed version of this notebook section by viewing [this notebook snapshot](https://terra-preprod-ui-terra.api.verily.com/workspaces/getting-started-with-workflows-workspace/resources/eebc2df6-fc9d-491e-874c-b87dbd3a68e1/notebook_snapshots/nextflow_examples_second_exercise_fully_executed.html). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d533789c-dc25-4f64-b5bd-78fe53d96ad1",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Install `nf-core` tool\n",
+    "\n",
+    "Run the cell below to install the [`nf-core` companion tool](https://nf-co.re/tools) in your Terra cloud environment, which will enable you to interact with `nf-core` pipelines via its command-line interface."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "881e8faa-a865-4fb3-ba69-7f4df579931a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import nf_core\n",
```

Comment:
Test this from a fresh notebook VM.  This is no longer working for me unless I restart the kernel. Apparently something has changed.
We probably should:
1. move this import to the top of the setup section (I was thinking to suggest that anyway)
2. split up the actions: do the install, then add a cell to restart the kernel (that's why we want this at the start of the setup)
3. then do the import

Reply:
I'm also seeing errors here. In my experience, package installs require a kernel restart before it can be imported. I'd recommend implementing one of Matt's or Amy's suggestions from the chat here. In particular, I'm fond of moving this import to the top, and then explicitly say "after running it, you'll likely need to restart your kernel".

Reply:
Thanks for updating this, Emma. Could you please move the imports all the way to the top? Right now, the BUCKET_REFERENCE cell is above the imports. So after I restart the kernel, I lose the BUCKET_REFERENCE variable.